### PR TITLE
Fix some dead links

### PR DESCRIPTION
**Pull request by RDIL**
## PR Summary
Now that I added a link check task, I'm going to fix all the links it detects as broken!  

  

## PR Context  

Dead links are bad links and bad links are bad documentation if that made any sense

## PR Checklist

- [X] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [X] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [X] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [X] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.  
- **User-facing changes**
    - [X] Not Applicable
    - **OR**  
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [X] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [X] [Add `[feature]` to your commit messages if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

**Comment by RDIL** [comment]
Static analysis come out wherever you are

**Comment by RDIL** [comment]
!wip refresh

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -354,7 +354,7 @@ Once you sign a CLA, all your existing and future pull requests will be labeled
 [up-for-grabs]: https://github.com/powershell/powershell/issues?q=is%3Aopen+is%3Aissue+label%3AUp-for-Grabs
 [semantic linefeeds]: http://rhodesmill.org/brandon/2012/one-sentence-per-line/
 [PowerShell-Docs]: https://github.com/powershell/powershell-docs/
-[use-vscode-editor]: ../docs/learning-powershell/using-vscode.md#editing-with-visual-studio-code
```
Comment (RDIL) [inline .github/CONTRIBUTING.md]: For some reason, this file doesn't exist anymore.

**Comment by RDIL** [comment]
macos failing question mark question mark

**Comment by TravisEz13** [comment]
I may have duplicated some of this work.  I created this PR to make the static analysis fail if there is a broken link:
https://github.com/PowerShell/PowerShell/pull/8653

**Comment by RDIL** [comment]
Yeah it seems that way.  I reviewed that PR.

**Pull request closed without merging**